src/entries/popup/pages/settings/networks.tsx

```diff
@@ -32,16 +44,25 @@ const chainLabel = ({ chainId }: { chainId: ChainId }) => {
 
 export function SettingsNetworks() {
   const navigate = useRainbowNavigate();
-  const {
-    userChains,
-    userChainsOrder,
-    updateUserChainsOrder,
-    updateUserChain,
-  } = useUserChainsStore();
+  const { userChainsOrder, updateUserChainsOrder } = useUserChainsStore();
   const supportedChains = getSupportedChains();
+
+  // This is the list of testnets that the user has added and there's no mainnet chain associated
+  // We still need to show them in the list
+  const testnetsOnly = useMemo(() => {
+    const { chains } = getNetwork();
+    return chains.filter(
+      (chain: Chain) =>
+        chain.testnet &&
+        !supportedChains.map((chain) => chain.id).includes(chain.id) &&
+        !SUPPORTED_CHAINS.map((chain) => chain.id).includes(chain.id),
+    );
+  }, [supportedChains]);
```

Comment:
we have `chainIdMap` to get the testnet chains related to a rnbw supported mainnet, maybe using that is simpler